src/build-index.js

```diff
@@ -249,7 +255,7 @@ async function runInfo(specs) {
 
     // Set the series title based on the info returned by the W3C API if
     // we have it, or compute the series title ourselves
-    const seriesInfo = specInfo.__series[spec.series.shortname];
+    const seriesInfo = specInfo.__series?.[spec.series.shortname];
```

Comment:
Needed when running a "diff build" that only has ISO specs as `__series` is only set when there are W3C specs.